test/test_custom_evented.rs

```diff
@@ -176,120 +197,6 @@ mod stress {
         }
     }
 
-    #[test]
-    fn multi_threaded_poll() {
```

Comment:
Is this test coming back?

Reply:
No since we are getting rid of multi threaded polling (removal of the locks).